build.properties

```diff
@@ -2,10 +2,8 @@
 # Don't directly modify this file. Instead, copy it to local.build.properties and
 # edit that.
 #
-# $Id$
 project.name = eXist-db
-project.version = 3.0
```

Comment:
`build` folder was `excluded` and that's why my search for `project.version.numeric` show nothing. I did commit updates to build instructions.